apps/src/blockly/constants.ts

```diff
@@ -96,6 +96,14 @@ export const PROCEDURE_DEFINITION_TYPES = [
 // A list of block types associated with the Run button.
 export const SETUP_TYPES = [BLOCK_TYPES.whenRun, BLOCK_TYPES.danceWhenSetup];
 
+export const WORKSPACE_EVENTS = {
+  MAIN_BLOCK_SPACE_CREATED: 'mainBlockSpaceCreated',
+  EVENT_BLOCKS_IMPORTED: 'blocksImported',
+  BLOCK_SPACE_CHANGE: 'blockSpaceChange',
+  BLOCK_SPACE_SCROLLED: 'blockSpaceScrolled',
+  RUN_BUTTON_CLICKED: 'runButtonClicked',
```

Comment:
It could be a good follow-up to audit these for use. I'd expect we're not using them all.